mu/logic.py

```diff
@@ -234,6 +234,56 @@ def check_pycodestyle(code):
     return style_feedback
 
 
+def get_runtime_hex_path():
```

Comment:
To keep similar things together would this not be better put after `get_workspace_dir` rather than `check_pycodestyle` (keep all the settings related functions together)